go-controller/pkg/ovn/ovn.go

```diff
@@ -180,8 +214,6 @@ type Controller struct {
 	retryEgressNodes *RetryObjs
 	// EgressIP Node-specific syncMap used by egressip node event handler
 	addEgressNodeFailed sync.Map
-	// Objects for nodes that need to be retried
-	retryNodes *RetryObjs
```

Comment:
Why did you move this higher up the struct's attribute list ?

Reply:
because most of RetryObjs are not for multi-network support. only retryPods/retryNodes are needed. Later, retryNetworkPolicies and retryNamespaces will be needed to multi-networkpolicies support.